crud/src/main/java/org/openmbee/sdvc/crud/controllers/projects/ProjectsController.java

```diff
@@ -112,6 +112,10 @@ public ProjectsResponse createOrUpdateProjects(
             }
 
             ProjectService ps = getProjectService(json);
+            if ((json.getProjectType()).equals("default")) {
```

Comment:
default or undefined is valid - look for the ProjectSchemas component in core and how it's used in the crud/cameo modules to register schemas - you'll need to inject that into this class in order to check, should check before the getProjectService call

you'll want to check everything still works by running the test (at the minimum crud test), and add to the crud test by trying to create a project with schema like 'invalid' and check the return code (you can modify the collection by importing it into postman, add a test, and export it again)

Reply:
i added a new commit with the modified check and crud.postman_collection